File: tundra-cli/benchmarks/summing/sum.go
// File: sum_squares/sum_squares.go
package main

import "fmt"

func sumSquares(n int) int64 {
    var total int64 = 0
    i := 1
    for i <= n {
        ii := int64(i) * int64(i)
        total += ii
        i++
    }
    return total
}

func main() {
    n := 3000000
    result := sumSquares(n)
    fmt.Println(result)
}

File: tundra-cli/benchmarks/summing/sum.rs
// File: sum_squares/sum_squares.rs
fn sum_squares(n: i32) -> i64 {
    let mut total: i64 = 0;
    let mut i: i32 = 1;
    while i <= n {
        let ii = (i as i64) * (i as i64);
        total += ii;
        i += 1;
    }
    total
}

fn main() {
    let n: i32 = 3_000_000;
    let result = sum_squares(n);
    println!("{}", result);
}
